overlay.d/07el9/etc/ssh/sshd_config.d/41-rhcos-defaults.conf

```diff
@@ -0,0 +1,8 @@
+# Disable PasswordAuthentication in SSH to preserve RHCOS 8 defaults
+# See: https://issues.redhat.com/browse/OCPBUGS-11613
+# See: https://github.com/openshift/os/issues/1216
+PasswordAuthentication no
```

Comment:
We have `50-disable-passwords.conf` in FCOS that also does this. Should we share it instead of redefining it here?